### PR TITLE
Frontend testing

### PR DESCRIPTION
**Pull request by Robert-Tice**
Frontend testing campaign.

### CODE REVIEW

**Comment by Robert-Tice** [comment]
Removed the carousel. Need to find a replacement.

**Comment by Robert-Tice** [comment]
This last commit changes the look of the index page. @gusthoff Let me know if you think the new content is ok.

File: content/index.rst [hunk 1]
```diff
@@ -11,8 +11,19 @@ LEARN.ADACORE.COM
...
+What is Ada and SPARK?
+-----------------------
 
-**Learn.adacore.com is an interactive learning platform designed to teach the Ada and SPARK programming languages.**
+Ada is a state-of-the art programming language that development teams worldwide
+are using for critical software: from microkernels and small-footprint,
+real-time embedded systems to large-scale enterprise applications, and
+everything in between.
+
+SPARK is formally analyzable subset of Ada — and toolset that brings
```
Comment (gusthoff) [inline content/index.rst:22]: Minor editorial comment: you can use `|mdash|` instead of the m-dash symbol.

**Comment by Robert-Tice** [comment]
Need to close this while resolving merge issues.

**Comment by Robert-Tice** [comment]
The latest commit has the package upgrade for the current dependabot alert.

**Pull request merged**